manuscript_analyses/collate_Figure5.R

```diff
@@ -0,0 +1,32 @@
+#compile figure panel
```

Comment:
for all R scripts, doesn't need to be changed, but typically would add spaces around operators and after commas